internal/pkg/manifest/lb_web_svc.go

```diff
@@ -70,6 +72,63 @@ type LoadBalancedWebServiceProps struct {
 	Platform    PlatformArgsOrString // Optional platform configuration.
 }
 
+// ExposedPorts returns all the ports that are container ports available to receive traffic.
+func (cfg *LoadBalancedWebService) ExposedPorts() ([]ExposedPort, error) {
```

Comment:
Is this identical to backend service, can we get rid of this duplication?
For example, a function that takes as input the image, http, and sidecar configurations.